### ISSUE TITLE
getaddressinfo not working in Knots

### ISSUE THREAD

**Issue by levantah** [bitcoinknots/bitcoin#77]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behaviour

a new in-knots created wallet is unable to `getaddressinfo` in Knots (26.2) but works well in Core (27.0rc1)

![image](https://github.com/bitcoinknots/bitcoin/assets/155245829/fb74943c-28f0-4d10-8876-c4b6f196cf50)

![image](https://github.com/bitcoinknots/bitcoin/assets/155245829/044819e1-1215-445f-a226-00f6d88f1a60)

### Expected behaviour

When the wallet is loaded*, the `getaddressinfo` should return a successful answer like it does on Bitcoin Core.

* sadly this RPC works only when a wallet is loaded

### Steps to reproduce

1. All this is on the Bitcoin mainnet.
1. Create a new wallet, name it `newlycreated`.
1. Call this: `getaddressinfo bc1qmxc2y0crywjvudp8cvhcfe3wx8mmgyvxw42xg3`,

### Relevant log output

```
Executing command using "newlycreated" wallet
getaddressinfo bc1qmxc2y0crywjvudp8cvhcfe3wx8mmgyvxw42xg3
Requested wallet does not exist or is not loaded (code -18)
```

### How did you obtain Bitcoin Knots

Pre-built binaries

### What version of Bitcoin Knots are you using?

v26.1.knots20240325

### Operating system and version

Ubuntu-compatible GNU libc Linux (Pop!_OS 22.04 LTS)

### Machine specifications

x86_64 Intel

```
$ uname -a
Linux pop-os 6.8.0-76060800daily20240311-generic #202403110203~1711393930~22.04~331756a SMP PREEMPT_DYNAMIC Mon M x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by luke-jr** [comment]
Should be fixed in 26.1.knots20240513, please confirm and close if so

**Comment by levantah** [comment]
Yes, works in the latest release. Thanks.
